spec/models/bookstore_spec.rb

```diff
@@ -19,4 +19,17 @@
 
   it { is_expected.to validate_presence_of :user }
   it { is_expected.to have_many(:reviews).dependent(:destroy) }
+
+  describe "#tweet" do
+    let!(:user) { FactoryGirl.create(:user) }
+    let!(:bookstore) { FactoryGirl.create(:bookstore, name: "Bookstore Awesome", city: "Boston", state: "Massachusetts") }
+
+     it "creates a tweet" do
+       VCR.use_cassette 'model/bookstore' do
+          response = bookstore.tweet
+          expect(response).to be_a(Twitter::Tweet)
+          expect(response.text).to include("#{bookstore.name}, #{bookstore.city}, #{bookstore.state} was just added!")
+       end
+     end
```

Comment:
A test for a case where the Twitter API fails on you for whatever reason should be included in this set of model tests.